ecli/src/config/mod.rs

```diff
@@ -63,7 +63,7 @@ impl ProgramConfigData {
             program_data_buf: prog_buf,
             extra_arg: args.extra_arg.clone(),
             btf_path: btf_dir_path,
-            prog_type: ProgramType::Undefine,
+            prog_type: args.prog_type.clone(),
```

Comment:
> It looks like we can put `args.get_file_content()` in an earlier function so that the arguments accepted by `async_try_from()` don't have to be clone.

`ProgramType` is an empty enum that does not contain any type, thus I think its clone is very cheap.

Reply:
LGTM